File: kyungyeul/algorithm1/201/단어뒤집기/word.py
n = list(input())
a = False
word = ''
result = ''
for i in n:
    if a == False:
        if i == '<':
            a = True
            word += i
        elif i == ' ':
            word += i
            result += word
            word = ''
        else:
            word += i
    elif a == True:
        word = word+i
        if i == '>':
            a = False
            result += word
            word = ''

print(result+word)

File: kyungyeul/algorithm1/200/단어뒤집기/word.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});
let input = [];
rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  const a = Number(input[0]);
  for (let i = 1; i <= a; i++) {
    let data = input[i].split(" ");
    let arr = [];
    for (let j in data) {
      let result = data[j].split("").reverse().join("");
      arr.push(result);
    }
    console.log(arr.join(" "));
  }
  process.exit();
});
